pkg/database/migrations.go

```diff
@@ -342,6 +342,18 @@ func (db *Database) getMigrations(ctx context.Context) *gormigrate.Gormigrate {
 				return nil
 			},
 		},
+		{
+			ID: "00014-DropUserDisabled",
+			Migrate: func(tx *gorm.DB) error {
+				logger.Infof("db migrations: dropping user disabled column")
+				sql := "ALTER TABLE users DROP COLUMN IF EXISTS disabled"
```

Comment:
Q: does this drop the column from the users_purge_index?

also, perhaps we don't need that index at all and should drop it here, if it exists.

Reply:
> When you remove a column from a table, PostgreSQL will automatically remove all of the indexes and constraints that involved the dropped column.

Seems so. Added a step to remove the index.